teams/deployment/membership.json

```diff
@@ -0,0 +1,44 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "DanielZhangQD",
```

Comment:
According to #527 , I think the membership file here should be updated also. cc @AstroProfundis .